app/models/invitation.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Invitation < ApplicationRecord
+  belongs_to :school
+  validates :email_address,
+            format: { with: EmailValidator.regexp, message: I18n.t('validations.invitation.email_address') }
+  validate :school_is_verified
+  after_create :send_invitation_email
```

Comment:
Minor: It _might_ be worth considering using `after_create_commit` vs `after_create` to ensure that we only ever queue an email for sending if the invitation record is successfully committed to the database...?

Reply:
Good idea, I've made that change in [3800844](https://github.com/RaspberryPiFoundation/editor-api/pull/312/commits/3800844f78d097be81f3c5bc4b0bc2bfc281954c)